spec/jobs/process_avatar_job_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe ProcessAvatarJob, type: :job do
+  subject { ProcessAvatarJob.new }
+
+  describe "#perform" do
+    let(:blob_storage) { double }
+    let(:user) { build(:user)  }
+    let(:file_path) { '/Users/blah/eicar' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.